include/ntptimeclient.h

```diff
@@ -39,8 +39,8 @@
 #include <mutex>
 
 // define the NTP server to connect too (replace . [dots] in IP addresses with , [commas])
-//#define cszNTPServer  192, 168, 1, 2
-#define cszNTPServer  94, 199, 173, 123     // 0.pool.ntp.org
+//#define cszNTPServer  94, 199, 173, 123     // 0.pool.ntp.org
+#define cszNTPServer  192, 168, 1, 2

```

Comment:
I'm thinking it may be better to point to a public NTP server in the main branch? The local IP address in this #define probably won't work for most repo users.